core/serialize_deserialize_test.js

```diff
@@ -63,12 +61,11 @@ function main() {
   ];
 
   assertArrayEquals(
-    Deno.core.opSync("op_serialize", circularObject),
+    Deno.core.ops.op_serialize(circularObject),
```

Comment:
`op_serialize` and `op_deserialize` are fallible.

Reply:
These are testing array equality of valid results, ie. here the calls will not throw => there's no need for the test to use `unwrapOpResult`.